config-service-impl/src/main/java/org/hypertrace/config/service/store/FilterBuilder.java

```diff
@@ -54,47 +61,34 @@ private Filter evaluateCompositeExpression(LogicalFilter logicalFilter) {
   }
 
   private Filter evaluateLeafExpression(RelationalFilter relationalFilter) {
+    Object value = this.convertValueToObject(relationalFilter.getValue());
```

Comment:
So are we sure the Value object doesn't work as-is here? What does doc store do with it?

Reply:
Got it, I had assumed they were stringifying it for some reason (and it stringifies to JSON). In that case, approach above makes sense.